sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ServiceVersion.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration;
+
+/**
+ * The versions of Azure App Configuration supported by this client library.
+ */
+public enum ServiceVersion {
+    V1_0("1.0");
```

Comment:
The service version should also be included in the user-agent string.

Reply:
Added `versionString` to user-agent string.